report.yaml

```diff
@@ -23,13 +23,14 @@ requests:
                 - Thriller
                 - Action
         - $group:
-            _id: $genres # Group data by the genre field
-              # Calculate the average Rotten Tomatoes viewer and critic ratings for each genre. 
+            _id:
```

Comment:
Maybe we can move the comment around so the yaml formatter doesn't do this weird thing

Reply:
Change made in 6b111d57a2af949644ffc98e42b35a0ded939c4a